<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formatting</class>
 <widget class="QWidget" name="formatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_yes_format" name="label_yes">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>是</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_backdrop">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>720</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>720</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>是否进行格式化？</string>
   </property>
  </widget>
  <widget class="Label_no_format" name="label_no">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_backdrop</zorder>
  <zorder>label_yes</zorder>
  <zorder>label_num</zorder>
  <zorder>label_no</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_no_format</class>
   <extends>QLabel</extends>
   <header>formatting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_yes_format</class>
   <extends>QLabel</extends>
   <header>formatting.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
